server/service/src/invoice/prescription/update/generate.rs

```diff
@@ -151,6 +151,9 @@ fn set_new_status_datetime(invoice: &mut InvoiceRow, status: &Option<UpdatePresc
     let status_datetime = get_invoice_status_datetime(invoice);
 
     match (&invoice.status, new_status) {
+        (InvoiceStatus::Verified, UpdatePrescriptionStatus::Cancelled) => {
+            invoice.cancelled_datetime = Some(status_datetime)
+        }
```

Comment:
This will *only* set the cancelled_datetime if "Cancelled" is being set *from* "Verified", but we don't actually have a check to prevent setting Cancelled from "Picked" (except in the front-end UI). This okay?